akka-actor-typed-tests/src/test/scala/akka/actor/typed/ActorContextSpec.scala

```diff
@@ -122,8 +122,12 @@ object ActorContextSpec {
           case Stop ⇒
             Behaviors.stopped
           case Kill(ref, replyTo) ⇒
-            if (ctx.stop(ref)) replyTo ! Killed
-            else replyTo ! NotKilled
+            try {
```

Comment:
I think you should refactor this test, maybe re-write it using our testkit :trollface:

Reply:
I'm saving it for you.